geode-cq/src/test/java/org/apache/geode/internal/cache/tier/sockets/ClientServerMiscBCDUnitTest.java

```diff
@@ -74,6 +75,15 @@ public ClientServerMiscBCDUnitTest(String version) {
     testVersion = version;
   }
 
+  @Override
+  void createClientCacheAndVerifyPingIntervalIsSet(String host, int port) throws Exception {
+    // this functionality was introduced in 1.5. If we let the test run in older
+    // clients it will throw a NoSuchMethodError
+    if (Version.CURRENT_ORDINAL >= 80 /* GEODE_150 */) {
```

Comment:
Could we just used `Version.GEODE_150` directly?

Reply:
Older versions of the product will run this code, so it can't refer to Version.GEODE_150.